easybuild/easyconfigs/o/OpenMPI/OpenMPI-4.1.6-NVHPC-24.1-CUDA-12.4.0-CUDAcore-12.4.0.eb

```diff
@@ -0,0 +1,72 @@
+name = 'OpenMPI'
+version = '4.1.6'
+versionsuffix = '-CUDAcore-12.4.0'
```

Comment:
Is there any specific reason for `CUDAcore`?

Reply:
No, I guess that was a typo from my side.